packages/app/src/systems/Core/machines/fetchMachine.test.ts

```diff
@@ -50,7 +50,7 @@ describe('fetchMachine', () => {
     expect(fetchSpy).toHaveBeenCalledTimes(3);
   });
 
-  it('should fail after attempting 5 times', async () => {
+  it.only('should fail after attempting 5 times', async () => {
```

Comment:
```suggestion
  it('should fail after attempting 5 times', async () => {
```

Reply:
would be nice to remove orthogonal changes to dependency updates, keeping this PR cleaner.

those changes can be done in other PR.
I see some breaking tests, maybe this changes are the reason ?

Reply:
This was failing after update deps, because of that I needed to fixed,